olive/evaluator/olive_evaluator.py

```diff
@@ -154,6 +155,9 @@ def evaluate(
     @staticmethod
     def fill_metric_user_config_with_model_io(metric: Metric, model: OliveModel):
```

Comment:
the method name should be updated?

Reply:
I prefer to `generate` as it won't change the original metrics.